sdks/python/container/py36/base_image_requirements.txt

```diff
@@ -68,7 +69,7 @@ google-cloud-videointelligence==1.16.1
 google-cloud-vision==1.0.0
 google-crc32c==1.3.0
 google-pasta==0.2.0
-google-python-cloud-debugger==2.18
+google-python-cloud-debugger==2.8
```

Comment:
Was this an accidental manual change?

Reply:
yeah there is something weird going on here. I've manually changed this PR to only add cloudpickle. It shouldn't be doing anything else.